plugins/scaffolder-backend-module-bitbucket-cloud/src/autocomplete/autocomplete.ts

```diff
@@ -34,10 +34,13 @@ export async function handleAutocompleteRequest({
 
   switch (resource) {
     case 'workspaces': {
-      const results: { title: string }[] = [];
+      const results: { title: string; id: string }[] = [];
 
       for await (const page of client.listWorkspaces().iteratePages()) {
-        const slugs = [...page.values!].map(p => ({ title: p.slug! }));
+        const slugs = [...page.values!].map(p => ({
+          title: p.slug!,
+          id: p.uuid!,
```

Comment:
So I think that we don't want to introduce an ID if we don't need it.

The ID should be the same as the title today, it's just that the title might be something other than the ID if there's a better representation of it.

Don't want to make any braking changes if we can avoid it. The ID would be used in the frontend as the `onChange` value, but the `title` would be whats rendered in dropdowns.

Does this make sense?

Reply:
I think it does.

Hopefully, I interpreted that correctly.